UI update v1

Mannyxp opened the pull request:
Completed Settings Screen and Tour Guide Screen UIs. Will update Tour Screen images when UI is more solidified.

Mannyxp committed:
UI update v1

Added Settings Screen and Tour Guide Screen UIs.

Mannyxp committed:
UI updates

Updated Home screen to reflect design, updated navigation, created missing screens, updated Ask Question screen.

zcappella commented:
We should make sure that future updates are using feature branches, even UI upgrades/overhauls

Mannyxp committed:
UI update fixes

Removed mention of Dream Team in design, hid Timeline feature button, added bottom nav bar on some screens in new style, changed back button color to be more visible,

vivekgoel1000 committed:
Settings screen test

Checks for three options in the Settings screen

laurahamann commented:
@Mannyxp can you resolve the merge conflicts? we should try to merge this in asap for testing purposes. I don't want this to get so far behind and difficult to merge

Mannyxp committed:
License and Policy Screen

Adding screens for now, still working on handling denying user policy.

Mannyxp committed:
Merge branch 'UI-Screens-v1' of https://github.com/umgc/fall2023 into UI-Screens-v1

Mannyxp committed:
Merge branch 'development' into UI-Screens-v1

Commit:
fix automated build issue and merge issue.

BenjaminSutter commented:
Since this is the initial iteration, I think we should try to make sure that all screens match the style with this update.

BenjaminSutter committed:
Simplified icon and font size

Updated the icon width/height and font size into a variable for easy adjustment

vivekgoel1000 committed:
Checking buttons on home screen

The new user interface has extra functionality

vivekgoel1000 committed:
Conversation History button check

Shouldn't break the build since the previous button has been forced into view

Mannyxp committed:
Completed Audio screen

Created Audio recording screen in new design

Mannyxp committed:
Update homeScreen.dart

Updated home screen based on meeting between Team A+B UI leads
